### PR TITLE
Add Ember Times Logo to Tag Page and Ember Times Posts

### LINKED ISSUE #3432: Include Ember Times Logo into the Blog for Ember Times Posts
The Ember Times now has a cool new logo that we could use for related posts on the blog and the [Newsletter overview page](https://www.emberjs.com/blog/tags/newsletter.html).

## Expected Behavior

Please include the Ember Times logo in the header of the following pages
- posts, which are tagged as `Newsletter`
- the overview tag pages for the Newsletter: https://www.emberjs.com/blog/tags/newsletter.html and https://www.emberjs.com/blog/tags/ember-times.html

## Sources

You can find the newsletter logo [here](https://www.emberjs.com/logos/).

### PR DESCRIPTION
**Pull request by jessica-jordan**
## What it does

This uses the Ember Times Logo and places it as a header item into the newsletter and Ember Times overview pages. It also adds the logo as a header item into each single Ember Times post that has been tagged as `Newsletter`.

To be discussed: On the tag page I decided to hide the heading `Newsletter` via `visibility:hidden` and include the image together with `Ember Times` as an `alt` attribute in the same `h1` tag.
If this isn't sufficient to provide a great a11y experience, I can improve it as suggested.
See also the screenshot and the screen recording below.

## To test locally:
Visit on the following routes on your local instance:
- Tag Pages: http://localhost:4567/blog/tags/newsletter.html and http://localhost:4567/blog/tags/ember-times.html
- Post Page, e.g. http://localhost:4567/blog/2018/06/29/the-ember-times-issue-53.html

## Related Issue(s)

Closes #3432

## Sources

| UI | screen-read w/ Voice Over | inspected w/ Firefox A11Y Inspector |
|--|--|--|
| embertimeslogoheader | https://streamable.com/tqgcl | screen shot 2018-06-30 at 11 04 01|

### CODE REVIEW

File: source/blog/tag.html.erb [hunk 1]
```diff
@@ -5,7 +5,16 @@ responsive: true
 ---
 <% @title = tagname %>
 
-<h1 class="text-left"><%= tagname %></h1>
+<% if tagname == 'Newsletter' || tagname == 'Ember Times' %>
+  <h1 class="text-left">
```
[the remark this answers is not in the document]
Comment (jessica-jordan) [reply source/blog/tag.html.erb]: Sure that sounds good - I can update that accordingly :+1:

**Comment by wifelette** [comment]
This is so great, thanks so much for all your hard work @jessica-jordan!

**Comment by jessica-jordan** [comment]
I just updated the PR accordingly :+1:

File: source/layouts/blog.erb [hunk 2]
```diff
@@ -2,9 +2,14 @@
 <% partial_for :sidebar, 'blog/sidebar' %>
 <% partial_for :foot, 'footer' %>
 <% wrap_layout :layout do %>
-  
+
   <article class="blog-post">
     <div class="blog-post-header">
+      <% if current_page.data.tags.include? 'Newsletter' %>
+        <a class="blog-header" href="<%= tag_path 'Newsletter' %>">
```
Comment (MelSumner) [inline source/layouts/blog.erb]: Please add aria-label="read all blog posts marked newsletter" or something similar- right now this is showing up as a blank link in the accessibility tree. Thank you!

Comment (jessica-jordan) [reply source/layouts/blog.erb]: Thank you for the heads up, I just read some more about it and I added the aria-label with a wording including `Ember Times` following a [suggestion on the WCAG working group Wiki to include a part of the original anchor text into the aria-label wording](https://www.w3.org/WAI/GL/wiki/Using_aria-label_for_link_purpose#Example_1:):

> [...] in that the simpler, non-descriptive [...] text on the page can be replaced with a more descriptive label of the link. The words [of the anchor text] are repeated in the aria-label (which replaces the original anchor text of "[Read more...]") to allow consistent communication between users.


screen shot 2018-07-03 at 01 04 44

**Pull request merged**